plus/tool/src/main/webapp/WEB-INF/templates/form.html

```diff
@@ -182,6 +182,21 @@ <h1><span th="#{plus.tool.tenant.title}">Tenant:<span>
     let inp = document.getElementById(id+'-boolean');
     inp.value = checked ? 'true' : 'false';
 }
+</script>
+
+<script th:inline="javascript">
+    function validateTenantForm() {
+        let issuerInput = document.getElementById("issuer");
+        let issuerValue = issuerInput.value.trim();
+        let issuerInputErrorMessage = /*[[#{plus.tenant.form.issuer.error.message}]]*/;
```

Comment:
You have commented on the error message?

Reply:
Hi Kunal, this is the way Earle showed me how to  inject messages into JS from the ```message.properties```.

https://github.com/sakaiproject/sakai/blob/master/site-manage/site-group-manager/src/main/webapp/WEB-INF/templates/fragments/javascript.html#L239

Some explanations from chatGPT:

In JavaScript, /* and \*/ are used to enclose multi-line comments. When used in this context within a Thymeleaf template, the comment block /* \*/ is being used as a special syntax for embedding Thymeleaf expressions within JavaScript code.